### PR TITLE
test | Gidk 3bj75g | Aditi | env difference fixed

### PR DESCRIPTION
**Pull request by aditi-tiwari17**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
https://app.clickup.com/t/3bj75g

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.scss [hunk 1]
```diff
@@ -653,7 +653,7 @@ span.icon-export-icon {
         background-color: $white;
         z-index: -1;
     }
-    .displayBlock {
+    .d-block {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/invoice/preview/invoice.preview.component.scss:656]: Is this class required if we have this class in global styles?

File: apps/web-giddh/src/app/invoice/proforma/proforma-list.component.scss [hunk 2]
```diff
@@ -392,7 +392,7 @@ span.icon-advance-filter {
         padding-top: 10px;
         padding-bottom: 0;
     }
-    .smallLoading {
+    .small-loading {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/invoice/proforma/proforma-list.component.scss:395]: Can we merge these two styles as they both are for the same selector?

File: apps/web-giddh/src/app/invoice/settings/invoice.setting.component.scss [hunk 3]
```diff
@@ -1,11 +1,11 @@
 @import "../../../assets/css/color.scss";
-.webookList li {
+.webook-list li {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/invoice/settings/invoice.setting.component.scss:2]: Please check if it is `webhook`

**Review by prateek-giddh — CHANGES_REQUESTED** [review]
Please check comments

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.scss (hunk 1 above)
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/invoice/preview/invoice.preview.component.scss:656]: not required, deleted @prateek-giddh

File: apps/web-giddh/src/app/invoice/proforma/proforma-list.component.scss (hunk 2 above)
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/invoice/proforma/proforma-list.component.scss:395]: no they are same selector but for different media query.

File: apps/web-giddh/src/app/invoice/settings/invoice.setting.component.scss (hunk 3 above)
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/invoice/settings/invoice.setting.component.scss:2]: checked

File: apps/web-giddh/src/app/invoice/preview/invoice.preview.component.scss (hunk 1 above)
Comment (aditi-tiwari17) [reply apps/web-giddh/src/app/invoice/preview/invoice.preview.component.scss:656]: not required, deleted

**Review by prateek-giddh — APPROVED** [review]

**Pull request merged**